python/src/aiconfig/editor/client/src/components/ConfigNameDescription.tsx

```diff
@@ -128,15 +128,21 @@ export default memo(function ConfigNameDescription({
           <Title
             ref={nameDisplayRef}
             onClick={onClickEdit}
-            className={!readOnly ? classes.hoverContainer : undefined}
+            className={
+              !readOnly ? `${classes.hoverContainer} hoverContainer` : undefined
+            }
           >
             {name}
           </Title>
           {description && (
             <div
               ref={descriptionDisplayRef}
               onClick={onClickEdit}
-              className={!readOnly ? classes.hoverContainer : undefined}
+              className={
+                !readOnly
+                  ? `${classes.hoverContainer} hoverContainer`
+                  : undefined
+              }
```

Comment:
not related to this diff but curious why we set the class name to undefined in readonly mode

Reply:
Just from the original implementation, when in readonly mode it doesn't need any extra styling since it should just render as regular text without any hover styles or border shown.